### PR TITLE
minimal boilerplate for react-fullstack-graphql

### PR DESCRIPTION
**Pull request by manjula91**

### CODE REVIEW

**Comment by DevanB** [comment]
This looks great! 💯

File: minimal/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "minimal",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-cache-inmemory": "^1.1.7",
+    "apollo-client": "^2.2.3",
+    "apollo-client-preset": "^1.0.8",
```
Comment (DevanB) [inline minimal/package.json]: If you use `apollo-client-preset`, there is no need to add `apollo-cache-inmemory`, `apollo-client`, and `apollo-link-http`. If you want to install those three dependencies, then you can remove the `apollo-client-preset` dependency.

File: minimal/src/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '../src/styles/index.css';
+import App from '../src/components/App';
+import { ApolloProvider } from 'react-apollo';
+import { HttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { ApolloClient } from 'apollo-client';
+
```
Comment (DevanB) [inline minimal/src/index.js]: In addition to the above comment, if you are using `apollo-client-preset` you can import `HttpLink`, `InMemoryCache`, and `ApolloClient` from it like:

```js
import { HttpClient, InMemoryCache, ApolloClient } from 'apollo-client-preset'
```

**Comment by manjula91** [comment]
Updated :)

**Comment by marktani** [comment]
Boom 💥 great work everyone 🙌

**Pull request merged**